pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -15,15 +15,15 @@
 , which
```

(to support above review item)
```suggestion
, which
, testers
, teams-for-linux
```